src/MongoDB.Driver/Encryption/LibMongoCryptControllerBase.cs

```diff
@@ -144,6 +143,18 @@ protected async Task<byte[]> ProcessStatesAsync(CryptContext context, string dat
             return result;
         }
 
+        protected void ThrowIfUnsupportedPlatform()
+        {
+            var currentOperatingSystem = OperatingSystemHelper.CurrentOperatingSystem;
+            if (currentOperatingSystem != OperatingSystemPlatform.Windows)
+            {
+#if NETSTANDARD1_5
+                // The non-windows operating systems have limitations on using socket and SslStream methods with .netstandard1.5
+                throw new PlatformNotSupportedException($"Field-level encryption is not supported on {currentOperatingSystem} with .netstandard1.5.");
```

Comment:
In English (as opposed to in code) let's use the normal `.NET Standard 1.5` usage:

    throw new PlatformNotSupportedException($"Field-level encryption is not supported on {currentOperatingSystem} with .NET Standard 1.5.");

Reply:
done